mkdocs.yml

```
@@ -0,0 +1,145 @@
+    - Core: api/core.md
+    - Context: api/context.md
+    - State: api/state.md
+    - Helpers: api/helpers.md
+    - Types: api/types.md
+  - Examples:
+    - examples/index.md
+    - FastAPI Integration: examples/fastapi-integration.md
+  - Advanced:
+    - advanced/index.md
+    - Performance: advanced/performance.md
+    - Troubleshooting: advanced/troubleshooting.md
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)